Packs/CortexResponseAndRemediation/Playbooks/playbook-CVE-2025-31324__-_SAP_NetWeaver_Visual_Composer.yml

```diff
@@ -0,0 +1,3732 @@
+  required: false
+  description: The playbook description for Rapid Breach Response layout.
+  playbookInputQuery:
+- key: EDLDomainBlocklist
+  value: {}
+  required: false
+  description: The EDL domain blocklist name.
+  playbookInputQuery:
+- key: BlockIndicatorsAutomatically
+  value:
+    simple: "True"
+  required: false
+  description: Whether to block the indicators automatically or not.
+  playbookInputQuery:
+- key: CollectedIndicatorsSeverity
+  value:
+    simple: Malicious
+  required: false
+  description: "The verdict of the collected indicators. Default is \"Malicious\". \nOther options can be \"Suspicious\" and \"Unknown\"."
+  playbookInputQuery:
+- key: RunXQLHuntingQueries
+  value:
+    simple: "False"
+  required: false
+  description: Whether to perform XQL hunting queries. Default is "False".
+  playbookInputQuery:
+- key: RelatedCVEs
+  value:
+    simple: CVE-2025-31324
+  required: false
+  description: The log4j related CVEs.
+  playbookInputQuery:
+- key: QRadarTimeRange
+  value:
+    simple: Last 1 DAYS
+  required: false
+  description: The time range for QRadar query.
+  playbookInputQuery:
+- key: UserVerification
+  value:
+    simple: "False"
+  required: false
+  description: "Possible values: True/False.\nWhether to provide user verification for blocking IPs. \n\nFalse - No prompt will be displayed to the user.\nTrue - The server will ask the user for blocking verification and will display the blocking list."
```

Comment:
"Whether to provide user verification for blocking IPs. \n\nFalse - No prompt will be displayed to the user.\nTrue - The server will ask the user for blocking verification and will display the blocking list."